Formula/mariadb@10.8.rb

```diff
@@ -36,7 +36,8 @@ class MariadbAT108 < Formula
   keg_only :versioned_formula
 
   # See: https://mariadb.com/kb/en/changes-improvements-in-mariadb-108/
-  deprecate! date: "2023-05-01", because: :unsupported
+  # and https://mariadb.org/about/#maintenance-policy
+  deprecate! date: "2023-05-20", because: :unsupported
```

Comment:
Updated to use official EOL date from https://mariadb.org/about/#maintenance-policy

Release series | Stable (GA) Date | End-of-life
--- | --- | ---
[10.8](https://mariadb.com/kb/en/changes-improvements-in-mariadb-108/) | 20 May 2022 | **20 May 2023**